tools/update_modules_check/update_modules_check.py

```diff
@@ -159,6 +159,37 @@ def get_sub_it_modules(modules, total_num, current_num):
     output = output[1:len(output)]
     print(output)
 
+
+# If the check passes, it succeeds, and if the check fails, it exits.
+def check_connector_complete_status(output_module):
```

Comment:
Where you used `check_connector_complete_status`? I think you need update `.github/workflows/backend.yml` and add github actions to use `check_connector_complete_status` to check the connector.